docs/developer-tools/ThrowHelper.md

```diff
@@ -0,0 +1,150 @@
+---
+title: ThrowHelper
+author: Sergio0694
+description: Helper methods to efficiently throw exceptions
+keywords: windows 10, uwp, windows community toolkit, uwp community toolkit, uwp toolkit, debug, net core, net standard
+dev_langs:
+  - csharp
+---
+
+# ThrowHelper
+
+The [ThrowHelper](https://docs.microsoft.com/dotnet/api/microsoft.toolkit.diagnostics.ThrowHelper) class is a helper type that can be used to efficiently throw exceptions. It is meant to support the [Guard](https://docs.microsoft.com/dotnet/api/microsoft.toolkit.diagnostics.guard) APIs, and it should primarily be used in situations where developers need fine-grained controls over the exception types being thrown, or over the exact exception message to include.
```

Comment:
We probably want the Guard api link here to link to our full documentation set here instead of the API docs? Then add a link to the Guard API docs as a 'Related Topic' section below?

Reply:
Not sure I understand this correctly, by "full documentation" you mean [this link](https://docs.microsoft.com/windows/communitytoolkit/)?

Reply:
I was talking about [this link](https://docs.microsoft.com/en-us/windows/communitytoolkit/developer-tools/guard) but you can use the relative path to the markdown file.

Reply:
Ah, sure! Done in https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs/pull/370/commits/d3d83a40f846fecd396bdfc661220fce34e0f006 😄